me.stonegate.tsacdop.yml

```diff
@@ -0,0 +1,36 @@
+app-id: "me.stonegate.tsacdop"
+runtime: org.gnome.Platform
+runtime-version: "40"
+sdk: org.gnome.Sdk
+command: tsacdop
+separate-locales: false
+finish-args:
+- "--share=ipc"
```

Comment:
you should enable --device=dri without that it complains on my wayland system

Reply:
Thanks, I will try and check if it take some effect.